File: CodingSample/DotNet/Fundamentals/DataTypes/PrimitiveTypeTest/DemoApp/Investment.cs
using System;

class Investment
{
    public static double FutureValue(double payment, int count, bool risk)
    {
        double i = risk ? 0.08 : 0.06;
        return (payment / i) * (Math.Pow(1 + i, count) - 1);
    }

}

File: CodingSample/Java/Core/DataTypes/UserDefTypeTest/demo1/Investment.java
class Investment {

    //instance fields
    private double payment;
    private int count;
    private boolean risk;

    //constructor
    public Investment(double amount, int years) {
        payment = amount;
        count = years;
        risk = false;
    }

    //instance method (receives 'this' argument)
    public void allowRisk(boolean yes) {
        risk = yes;
    }

    public double totalPayment() {
        return count * payment;
    }

    public double futureValue() {
        double i = risk ? 0.08 : 0.06;
        return (payment / i) * (Math.pow(1 + i, count) - 1);
    }

}
